sqlglot/dialects/dremio.py

```diff
@@ -100,10 +100,13 @@ class Parser(parser.Parser):
         FUNCTIONS = {
             **parser.Parser.FUNCTIONS,
             "TO_CHAR": lambda args: (
-                exp.ToChar(this=args[0], format=args[1])
+                exp.Cast(this=args[0], to=exp.DataType.build("VARCHAR"))
                 if len(args) == 2 and (not isinstance(args[1], exp.Literal) or "#" in args[1].name)
                 else build_formatted_time(exp.TimeToStr, "dremio")(args)
             ),
+            "DATE_FORMAT": build_formatted_time(exp.TimeToStr, "dremio"),
+            "TO_DATE": build_formatted_time(exp.TsOrDsToDate, "dremio"),
+            "TO_TIMESTAMP": build_formatted_time(exp.TimeStrToTime, "dremio"),
```

Comment:
@jasonthomassql can we please revert these and add in a followup PR? Ideally the scope of each one should be the minimum possible.